src/main/java/wooteco/subway/exception/PageControllerAdvice.java

```diff
@@ -0,0 +1,20 @@
+package wooteco.subway.exception;
+
+import static org.springframework.http.HttpStatus.*;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class PageControllerAdvice {
+    @ExceptionHandler(IllegalArgumentException.class)
+    public ResponseEntity<String> illegalArgumentExceptionHandle() {
+        return ResponseEntity.badRequest().build();
+    }
+
+    @ExceptionHandler(RuntimeException.class)
+    public ResponseEntity<String> runtimeExceptionHandle() {
```

Comment:
```suggestion
    public ResponseEntity<Void> runtimeExceptionHandle() {
```
body가 없으면 Void로 명시해줄 수 있답니다. 😏